src/lib/functions/runtimes/js/builders/zisi.mjs

```diff
@@ -30,9 +31,6 @@ const buildFunction = async ({ cache, config, directory, func, hasTypeModule, pr
   }
   const functionDirectory = path.dirname(func.mainFile)
 
-  // performance
```

Comment:
this mentions performance as the reason we're late-importing it, so we should double-check wether we want to make it static.